src/config/ecosystem.ts

```diff
@@ -71,13 +72,12 @@ const getEcosystemId = async (): Promise<string> => {
 	if (!ecoId || !ecoRole) {
 		try {
 			if (orgId) {
-				const { data } = (await getEcosystems(orgId)) as AxiosResponse;
-
+				const { data } = (await getEcosystems(orgId, 1, 10, '')) as AxiosResponse;
```

Comment:
If the value of the 2nd, 3rd, and 4th argument is the default then you can remove it from here and add it into the function declaration code as a default value.

Reply:
resolved